server/src/main.rs

```diff
@@ -89,7 +82,7 @@ fn main() {
         // The errors are consciously ignored as the program is terminating.
         let binary_path = std::env::current_exe().unwrap();
         let binary_name = binary_path.file_name().unwrap();
-        error!(" Usage: {} data_folder", binary_name.to_str().unwrap());
+        println!(" Usage: {} data_folder", binary_name.to_str().unwrap());
```

Comment:
There should not be a space in the beginning of this message. Also should `println!` be used or `eprintln!`?

Reply:
Also as far as i can see, punctuation has only been added in `catalog.rs`. You might as well change all log messages that have been added in this branch.

Reply:
Fixed

Reply:
> There should not be a space in the beginning of this message. Also should `println!` be used or `eprintln!`?

I couldn't tell you the difference between stderr and stdout, I don't know if it is important to the program. Both options print it in the console fine

Reply:
I think `eprintln!` is correct here.

Reply:
Fixed

Reply:
By keeping normal output in stdout and stderr you can pipe normal output and errors to different places.